src/MoBi.Core/Domain/Services/SimulationFactory.cs

```diff
@@ -28,13 +28,16 @@ public class SimulationFactory : ISimulationFactory
       private readonly ICreationMetaDataFactory _creationMetaDataFactory;
       private readonly ISimulationParameterOriginIdUpdater _simulationParameterOriginIdUpdater;
       private readonly IDiagramManagerFactory _diagramManagerFactory;
+      private readonly ISimulationConfigurationFactory _simulationConfigurationFactory;
 
-      public SimulationFactory(IIdGenerator idGenerator, ICreationMetaDataFactory creationMetaDataFactory, ISimulationParameterOriginIdUpdater simulationParameterOriginIdUpdater, IDiagramManagerFactory diagramManagerFactory)
+      public SimulationFactory(IIdGenerator idGenerator, ICreationMetaDataFactory creationMetaDataFactory, ISimulationParameterOriginIdUpdater simulationParameterOriginIdUpdater, 
```

Comment:
all on one line I would say

Reply:
Are you sure? That's almost 300 characters long.

Reply:
Sorry what I mean is 
ONE PARAMETER PER LINE
So split at ,